src/cmd-core.h

```diff
@@ -92,6 +92,9 @@ typedef enum cmd_code {
 	CMD_OPEN,
 	CMD_CLOSE,
 	CMD_RUN,
+	CMD_EXPLORE,
+	CMD_NAVIGATE_UP,
```

Comment:
CMD_NAVIGATE_UP and CMD_NAVIGATE_DOWN are no longer used in cmd-core.c so I'd either remove them here or add them back in cmd-core.c.  I prefer the latter:  that allows other code to use something like cmdq_push(CMD_NAVIGATE_UP) and is consistent with how do_cmd_navigate_up() and do_cmd_navigate_down() are named and their placement in cmds.h and cmd-cave.c.